test/cases/loaders/esm-loader-type/test.filter.js

```diff
@@ -0,0 +1,5 @@
+module.exports = function(config) {
+	// TODO need fix in v8 https://github.com/nodejs/node/issues/35889
+	// TODO otherwise this test case cause segment fault
+	return false;
```

Comment:
maybe we could filter this spawning another `jest` process and catching exit code.. but it seems overengineering to me..

Reply:
I think yes